trader.js

```diff
@@ -187,27 +187,32 @@ socket.on("buy_signal", async (signal) => {
                     pair: signal.pair,
                     qty: Number(user_payload[tresult].buy_amount),
                 }
-                socket.emit("traded_buy_signal", traded_buy_signal)
                 ////
                 if (user_payload[tresult].trading_type === "real") {
                     bnb_client.mgMarketBuy(
                         "BTCUSDT",
                         Number(user_payload[tresult].buy_amount),
                         (error, response) => {
-                            if (error)
+                            if (error) {
                                 console.log(
                                     "ERROR 3 BTCUSDT ",
                                     Number(user_payload[tresult].buy_amount),
                                     error.body
                                 )
-                            if (response)
+                            }
+                            if (response) {
```

Comment:
I think we can keep this in Else condition of if(error) statement

Reply:
Well, the thing is: The code uses these a bit weird error/response checks.

I would argue that this PR should only move the emits (at is does) and not change the logic of the code otherwise. If you want to clear up the api success/error handling, you can do another PR

Reply:
(Or you can write a new issue if you believe this to be an issue)

Reply:
@mmarczuk please remove the `if (response)` checks, keeping only their body, and return at the end of the `if (error)` bodies, as the `response` variable is never actually used.

Reply:
@mmarczuk please read my previous message again carefully 😉 I didn't say "replace the second `if` with `else`", but "return at the end of the first error-`if`'s body; and remove the response-`if`.